Index: firefox-trunk-35.0~a1~hg20140919r206162/testing/testsuite-targets.mk
===================================================================
--- firefox-trunk-35.0~a1~hg20140919r206162.orig/testing/testsuite-targets.mk	2014-09-19 17:19:29.231205154 +0100
+++ firefox-trunk-35.0~a1~hg20140919r206162/testing/testsuite-targets.mk	2014-09-19 17:21:58.607205930 +0100
@@ -402,9 +402,11 @@
   stage-marionette \
   stage-cppunittests \
   stage-jittest \
-  stage-steeplechase \
   stage-web-platform-tests \
   $(NULL)
+ifdef MOZ_WEBRTC
+package-tests: stage-steeplechase
+endif
 else
 # This staging area has been built for us by universal/flight.mk
 PKG_STAGE = $(DIST)/universal/test-stage
