Index: firefox-46.0+build5/browser/locales/Makefile.in
===================================================================
--- firefox-46.0+build5.orig/browser/locales/Makefile.in
+++ firefox-46.0+build5/browser/locales/Makefile.in
@@ -62,10 +62,6 @@ STUB_HOOK = $(NSINSTALL) -D '$(ABS_DIST)
 endif
 
 SEARCHPLUGINS_NAMES = $(shell cat $(call MERGE_FILE,/searchplugins/list.txt)) ddg
-ifeq (,$(filter-out en-US ru be kk tr uk zh-CN zh-TW,$(AB_CD)))
-    SEARCHPLUGINS_NAMES := $(subst google,google:hidden,$(SEARCHPLUGINS_NAMES))
-    SEARCHPLUGINS_NAMES += google-nocodes
-endif
 SEARCHPLUGINS_FILENAMES = $(subst :hidden,,$(SEARCHPLUGINS_NAMES))
 SEARCHPLUGINS_PATH := .deps/generated_$(AB_CD)
 SEARCHPLUGINS_TARGET := libs searchplugins
